source/standards/storing-credentials.html.md.erb

```diff
@@ -0,0 +1,48 @@
+---
+title: How to store credentials
+last_reviewed_on: 2021-08-06
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+Depending on how you [manage your accounts](/standards/accounts-with-third-parties.html), you, your team and the service you run may have credentials or other secrets that you need to store securely.
+
+## Personal credentials
+
+Examples include login details for:
+
+- GitHub
+- AWS
+- GOV.UK Signon
+
+[Use the password manager built into to your browser in the first instance](https://lock.cmpxchg8b.com/passmgrs.html#conclusion).
+
+If you can't do this for some reason, then it's OK to use a third-party password manager. This could be necessary if your browser has an accessibility issue, or if you work with multiple browsers.
+
+Some of the third-party password managers people use at GDS include:
+
+- **LastPass** - Approved by Information Assurance and available to install in the GDS Self Service app. However, you can only use the free version on one type of device.
+- [**1Password**](https://1password.com/)
+- [**BitWarden**](https://bitwarden.com/) - An open source password manager.
+- [**KeePassXC**](https://keepassxc.org/) -  An offline password store, which you may want to backup somewhere.
+
+**Do not use browser extensions that autofill credentials**, as [these are fundamentally vulnerable to attack](https://lock.cmpxchg8b.com/passmgrs.html#interprocess-communication). If you do use an extension, make sure it's configured so that you manually copy / paste credentials.
```

Comment:
This is quite a strong stance for us to take. I imagine there are lots of people at GDS (myself included, although I'm changing that now) who use browser extensions which autofill passwords.

I understand and mostly agree with the arguments in Tavis' article. But if we really want to take the "autofill extensions considered harmful" stance at GDS, I think we probably need to do more work than just publishing a piece of guidance. We'd need a whole comms campaign to get people to change their behaviour. Which we'd need to be very careful didn't do more harm than good (I wouldn't want misinformation like "GDS say password managers are bad" to spread).

As an intermediate step, perhaps we could water this down to a warning, like:

> ⚠️  Warning! Browser extensions which autofill credentials have [serious security concerns](https://lock.cmpxchg8b.com/passmgrs.html#interprocess-communication). Carefully consider the risks before using a browser extension. If do use an extension, consider configuring so that you manually copy / paste credentials instead of allowing it to autofill them.

Reply:
Good points, Richard. Agreed - we should be careful about misinformation that could stem from this.

Reply:
Thanks @richardTowers. I agree we need to make a standard that reflects what we actually do, and strike a balance between instruction and information to let people decide for themselves.

I've added a new commit (e838bb834a512643f692d1cb63a1ae7d5752052b) with your suggestion, except the second sentence: it's unclear exactly what we're asking people to do.

How does it look now?

Reply:
Yep, that looks good to me.